tests/api/item_types/test_item_types_rest.py

```diff
@@ -125,7 +125,7 @@ def test_item_types_post_put_delete(client, org_martigny,
         headers=json_header
     )
     assert res.status_code == 200
-    # assert res.headers['ETag'] != '"{}"'.format(librarie.revision_id)
+    # assert res.headers['ETag'] != f'"{librarie.revision_id}"'
```

Comment:
To remove?

Reply:
This exists in multiple places, so I have left it alone.